src/ILCompiler.ReadyToRun/src/Compiler/ReadyToRunTableManager.cs

```diff
@@ -44,6 +59,24 @@ public IEnumerable<TypeDesc> GetTypesWithAvailableTypes()
             return _typesWithAvailableTypesGenerated;
         }
 
+        public IEnumerable<ExportedTypeInfo> GetExportedTypes()
+        {
+            List<ExportedTypeInfo> exportedTypeList = new List<ExportedTypeInfo>();
```

Comment:
This method could just do yield returns - building a list that we'll enumerate once and throw out is just giving the GC useless work.

Reply:
Good point, fixed in 2nd commit.